pkg/controller/crdmirroring/types/types.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+const (
+	ManagedByController = "crd.antrea.io/managed-by-mirroring-controller"
+	ControllerName      = "crdmirroring-controller"
+
+	StopMirror = "crd.antrea.io/stop-mirror"
+	Stopped    = "true"
```

Comment:
Should this be removed?

Reply:
This value is used in test code.
```go
func (c *ClusterGroupTestHandler) LegacyStaleAndWait(obj metav1.Object) (metav1.Object, metav1.Object, error) {
	obj.SetAnnotations(map[string]string{types.StopMirror: types.Stopped})
	res1, err := c.legacyClient.CoreV1alpha2().ClusterGroups().Update(context.TODO(), obj.(*legacycore.ClusterGroup), metav1.UpdateOptions{})
	if err != nil {
		return nil, nil, err
	}

	res2, err := c.waitForNewStale(obj.GetName(), timeout)
	if err != nil {
		return nil, nil, err
	}
	return res1, res2, nil
}
```

Reply:
We don't set value in main code, why it must be set in testing?

Reply:
Got it, I'll delete it.